synapse/lib/kv.py

```diff
@@ -55,10 +55,11 @@ def set(self, prop, valu):
         if self.vals.get(prop) == valu:
             return
 
-        self.vals[prop] = valu
+        byts = s_msgpack.en(valu)
+        self.vals[prop] = s_msgpack.un(byts)
```

Comment:
Is this faster or slower than comparing a deepcopy?

Reply:
My timeit test suggests that unmsgpack-ing the bytes of a small dict `{'a': 4, 'b': 5}` is about 1/3rd the time for a deepcopy.